src/CSnakes.SourceGeneration/Parser/PythonParser.TypeDef.cs

```diff
@@ -25,60 +25,116 @@ private static TokenListParser<PythonToken, PythonTypeSpec> CreatePythonTypeDefi
     {
         var typeDefinitionParser = Parse.Ref(() => PythonTypeDefinitionParser);
 
+        var comma = Token.EqualTo(PythonToken.Comma);
+
         var parser =
             from name in Parse.OneOf(Token.EqualTo(PythonToken.None),
                                      Token.EqualTo(PythonToken.Identifier),
                                      Token.EqualTo(PythonToken.QualifiedIdentifier))
             from type in name.ToStringValue() switch
             {
-                "None" => Parse.Return<PythonToken, PythonTypeSpec>(PythonTypeSpec.None),
-                "Callable" and var n =>
+                "None" => Return((PythonTypeSpec)PythonTypeSpec.None),
+                "Any" => Return((PythonTypeSpec)PythonTypeSpec.Any),
+                "int" => Return((PythonTypeSpec)PythonTypeSpec.Int),
+                "str" => Return((PythonTypeSpec)PythonTypeSpec.Str),
+                "float" => Return((PythonTypeSpec)PythonTypeSpec.Float),
+                "bool" => Return((PythonTypeSpec)PythonTypeSpec.Bool),
+                "bytes" => Return((PythonTypeSpec)PythonTypeSpec.Bytes),
+                "Buffer" or "collections.abc.Buffer" => Return((PythonTypeSpec)PythonTypeSpec.Buffer),
+                "Optional" or "typing.Optional" =>
+                    typeDefinitionParser.Subscript(PythonTypeSpec (of) => new OptionalType(of)),
+                "list" or "List" or "typing.List" =>
+                    typeDefinitionParser.Subscript(PythonTypeSpec (of) => new ListType(of)),
+                "Sequence" or "collections.abc.Sequence" or "typing.Sequence" =>
+                    typeDefinitionParser.Subscript(PythonTypeSpec (of) => new SequenceType(of)),
+                "dict" or "Dict" or "typing.Dict" =>
+                    typeDefinitionParser.Subscript(PythonTypeSpec (k, v) => new DictType(k, v)),
+                "Mapping" or "collections.abc.Mapping" or "typing.Mapping" =>
+                    typeDefinitionParser.Subscript(PythonTypeSpec (k, v) => new MappingType(k, v)),
+                "Generator" or "collections.abc.Generator" or "typing.Generator" =>
```

Comment:
How would we implement https://github.com/tonybaloney/CSnakes/issues/681 in this case?